### PR TITLE
Makefile/README updates

### LINKED ISSUE #74: Building Package from fresh - errors
Thumbs up on all of this...  just in case you don't know, a fresh run of your installation instructions results in these errors (at least for me.)

[PointFreeCo Build Errors.rtfd.zip](https://github.com/pointfreeco/pointfreeco/files/1590077/PointFreeCo.Build.Errors.rtfd.zip)

Thanks!

### PR DESCRIPTION
**Pull request by stephencelis**
Hopefully fixes #74.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,14 +1,21 @@
+bootstrap: check-postgres postgres-mm webkit-snapshot-mm init-db xcodeproj
```
Comment (stephencelis) [inline Makefile]: Keeping our default `make` command first makes it easy to link to should we want to.

File: Makefile [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
 	@testable import StyleguideTests;
 
-bootstrap: xcodeproj postgres-mm db
-
 xcodeproj:
 	swift package generate-xcodeproj --xcconfig-overrides=Development.xcconfig
 	xed .
 
-sourcery: linux-main route-partial-iso
+# db
```
Comment (stephencelis) [inline Makefile]: Adding some comment-delineated sections.

File: Makefile [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
+
+check-postgres:
```
Comment (stephencelis) [inline Makefile]: Quick bootstrap health check.

File: README.md [hunk 4]
```diff
@@ -8,25 +8,42 @@ This repo contains the application code for the [Point-Free](http://www.pointfre
...
+The repo contains an extensive test suite and some playgrounds to explore. To get things running:
 
-* `git clone https://github.com/pointfreeco/pointfreeco.git`
-* `cd pointfreeco`
-* `swift package generate-xcodeproj`
-* `xed .`
-* Run tests: Command+U
-* Build: Command+B
-* Open a playground!
+  * Make sure [Xcode](https://developer.apple.com/xcode/) is installed!
```
Comment (stephencelis) [inline README.md]: I've indented bullets by 2 spaces to make multi-line bullets easier to format.

File: Makefile (hunk 1 above)
Comment (mbrandonw) [inline Makefile]: we should prob also add the crypto module map from pointfreeco

**Pull request merged**